pkg/api/apis/operators/v1alpha2/operatorgroup_types.go

```diff
@@ -17,10 +17,20 @@ type OperatorGroupSpec struct {
 
 	// ServiceAccount to bind OperatorGroup roles to.
 	ServiceAccount corev1.ServiceAccount `json:"serviceAccount,omitempty"`
+
+	// Static tells OLM not to update the OperatorGroup's providedAPIs annotation
+	// +optional
+	StaticProvidedAPIs bool `json:"staticProvidedAPIs,omitempty"`
```

Comment:
`StaticProvidedAPIs` helps ensure OperatorGroups like the one CMO will be using are not altered by OLM control loops.